source/code-snippets/usage-examples/deleteMany.js

```diff
@@ -0,0 +1,36 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+    // Query for all movies with the title "Santa Claus"
+    const query = { title: "Santa Claus" };
+
+    const options = {
```

Comment:
Let's omit these options as we would rather these are set at the database or collection level